src/riak_core_app.erl

```diff
@@ -145,3 +146,12 @@ register_capabilities() ->
       end,
       Capabilities),
     ok.
+
+%% TODO: This belongs in riak_kv - an issue will be created to move it, but time constraints.
```

Comment:
Understood, and that also answers another question I had about why `riak_core` depends on `eleveldb`.